team.go

```diff
@@ -343,25 +302,56 @@ func (client *Client) GetTeamCount() (int, error) {
 	return int(q.Account.Teams.TotalCount), HandleErrors(err, nil)
 }
 
-func (client *Client) ListTeams() ([]Team, error) {
+func (client *Client) ListTeams(variables *PayloadVariables) (TeamConnection, error) {
 	var q struct {
 		Account struct {
 			Teams TeamConnection `graphql:"teams(after: $after, first: $first)"`
 		}
 	}
-	v := client.InitialPageVariables()
-	return q.Account.Teams.Query(client, &q, v)
+	if variables == nil {
+		variables = client.InitialPageVariablesPointer()
+	}
+
+	if err := client.Query(&q, *variables); err != nil {
+		return TeamConnection{}, err
+	}
+
+	for q.Account.Teams.PageInfo.HasNextPage {
+		(*variables)["after"] = q.Account.Teams.PageInfo.End
+		resp, err := client.ListTeams(variables)
+		if err != nil {
+			return TeamConnection{}, err
+		}
+		q.Account.Teams.Nodes = append(q.Account.Teams.Nodes, resp.Nodes...)
+		q.Account.Teams.PageInfo = resp.PageInfo
+	}
+	return q.Account.Teams, nil
 }
 
-func (client *Client) ListTeamsWithManager(email string) ([]Team, error) {
+func (client *Client) ListTeamsWithManager(email string, variables *PayloadVariables) (TeamConnection, error) {
 	var q struct {
 		Account struct {
 			Teams TeamConnection `graphql:"teams(managerEmail: $email, after: $after, first: $first)"`
 		}
 	}
-	v := client.InitialPageVariables()
-	v["email"] = graphql.String(email)
-	return q.Account.Teams.Query(client, &q, v)
+	if variables == nil {
+		variables = client.InitialPageVariablesPointer()
+	}
+
+	if err := client.Query(&q, *variables); err != nil {
+		return TeamConnection{}, err
+	}
+
+	for q.Account.Teams.PageInfo.HasNextPage {
+		(*variables)["after"] = q.Account.Teams.PageInfo.End
+		resp, err := client.ListTeams(variables)
+		if err != nil {
+			return TeamConnection{}, err
+		}
+		q.Account.Teams.Nodes = append(q.Account.Teams.Nodes, resp.Nodes...)
+		q.Account.Teams.PageInfo = resp.PageInfo
```

Comment:
same total count thing
```suggestion
		q.Account.Teams.PageInfo = resp.PageInfo
		q.Account.Teams.TotalCount += resp.TotalCount
```